package.json

```diff
@@ -73,7 +73,7 @@
     "multihashes": "^0.2.2",
     "node-polyglot": "^2.0.0",
     "open": "0.0.5",
-    "select2": "^4.0.3",
+    "select2": "4.0.3",
```

Comment:
Even though we are using a modified source of the select2 JS, I left it in NPM because we're still referencing the CSS file from there. Now that I think about it, I think we should just self-host the CSS as well. Otherwise, there's a risk that a developer will import and use the NPM select2 JS instead of our MacGyver'd version.